charts/vald-benchmark-operator/Chart.yaml

```diff
@@ -32,7 +32,7 @@ appVersion: "1.16.0"
 
 apiVersion: v2
 name: vald-benchmark-operator
-version: v1.7.5
+version: v1.7.11
```

Comment:
version needs to be updated by ci like below
https://github.com/vdaas/vald/blob/9b088e3db26724f4a6f37e45e1ab6b5633f8982b/.github/workflows/_release-pr.yml#L77